prov/efa/configure.m4

```diff
@@ -13,6 +13,8 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 	efa_h_enable_poisoning=0
 	# Determine if efa device supports extensible CQ with access to additional fields from WC
 	efadv_support_extended_cq=1
+	# Determine if efa device support in-order write at 128 bytes aligned boundary.
+	efa_support_ll128=1
```

Comment:
small nit: `ll123` is visually confusing.  Can we use `LL128` ?

Reply:
ll128 is a term used by NCCL. We should use the term from rdma-core,